docs-samples/data-science/semantic-link-samples/data_cleaning_functional_dependencies_tutorial.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Tutorial: Clean data with functional dependencies\n",
     "\n",
-    "In this tutorial, you use functional dependencies for data cleaning. A functional dependency exists when one column in a dataset is a function of another column. For example, a _zip code_ column might determine the values in a _city_ column. A functional dependency manifests itself as a one-to-many relationship between the values in two or more columns within a DataFrame. This tutorial uses the _Synthea_ dataset to show how functional relationships can help to detect data quality problems.\n"
+    "In this tutorial, you use functional dependencies for data cleaning. A functional dependency exists when one column in a semantic model is a function of another column. For example, a _zip code_ column might determine the values in a _city_ column. A functional dependency manifests itself as a one-to-many relationship between the values in two or more columns within a DataFrame. This tutorial uses the _Synthea_ dataset to show how functional relationships can help to detect data quality problems.\n"
```

Comment:
```suggestion
    "In this tutorial, you use functional dependencies for data cleaning. A functional dependency exists when one column in a semantic model (a Power BI dataset) is a function of another column. For example, a _zip code_ column might determine the values in a _city_ column. A functional dependency manifests itself as a one-to-many relationship between the values in two or more columns within a DataFrame. This tutorial uses the _Synthea_ dataset to show how functional relationships can help to detect data quality problems.\n"
```

Reply:
because this is new terminology, it's probably a good idea to make it clear what it means the first time we mention it